.github/workflows/report_quiz/main.sh

```diff
@@ -0,0 +1,161 @@
+#!/bin/bash
+
+set -e
+
+BASE_DIR=$(dirname "${BASH_SOURCE[0]}")
+DATABASE_DIR="${BASE_DIR}/database"
+ARCHIVE_DIR="${BASE_DIR}/archive"
+TEMPFILE="$(mktemp)"
+
+# load functions
+source "${BASE_DIR}/common/utils.sh"
+source "${BASE_DIR}/quiz/functions.sh"
+source "${BASE_DIR}/leaderboard/functions.sh"
+
+# load environement variables
+source "${BASE_DIR}/env.sh"
+
+mkdir -p "${BASE_DIR}/database"
+
+if [ -z ${TELEGRAM_BOT_TOKEN} ]; then
```

Comment:
nitpick: 
```suggestion
[ -z ${TELEGRAM_BOT_TOKEN} ] && echo "...";
```

same for the others